docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,757 @@
+[role="xpack"]
+[[ootb-ml-jobs]]
+== Modules with prebuilt {anomaly-detect} jobs
+
+This page lists the available modules with prebuilt {anomaly-detect} jobs. Every 
+module item in the list contains a name, a short description, information about 
+the {ml-docs}/ml-buckets.html#ml-buckets[`bucket_span`] and the 
+{ml-docs}/ml-functions.html[analysis function] as well as the influencers – the 
+entities that have contributed to, or are to blame for, the anomalies – of the 
+{anomaly-detect} job.
+
+
+[float]
+[[ootb-ml-jobs-apache]]
+=== Apache
+
+low_request_rate_ecs::
+Detects low request rates in HTTP access log.
++
+--
+Bucket span: 15m.
+
+Function: `low_count`.
+--
+
+source_ip_request_rate_ecs::
+Detects source IPs with unusually high request rates in the HTTP access log.
++
+--
+Bucket span: 1h.
+
+Function: `high_count`.
+
+Influencers:
+
+* `source.address`
+--
+
+source_ip_url_count_ecs::
+Detects source IPs with unusually high distinct count of URLs in the HTTP access 
+log.
++
+--
+Bucket span: 1h.
+
+Function: `high_distinct_count`.
+
+Influencers:
+
+* `source.address`
+--
+
+status_code_rate_ecs::
+Detects unusual status code rates in the HTTP access log.
++
+--
+Bucket span: 15m.
+
+Function: `count`.
+
+Influencers:
+
+* `http.response.status_code` 
+* `source.address`
+--
+
+visitor_rate_ecs::
+Detects unusual visitor rates in the HTTP access log.
++
+--
+Bucket span: 15m.
+
+Function: `non_zero_count`.
+--
+
+[float]
+[[ootb-ml-jobs-apm]]
+=== APM
+
+abnormal_span_durations_jsbase::
+abnormal_span_durations_nodejs::
+Searches for spans that are taking longer than usual to process.
++
+--
+Bucket span: 15m.
+
+Function: `high_mean`.
+
+Influencers:
+
+* `service.name` 
+* `span.name`
+* `span.type` 
+* `trace.id`
+--
+
+abnormal_trace_durations_nodejs::
+Identifies trace transactions that are processing more slowly than usual.
++
+--
+Bucket span: 15m.
+
+Function: `high_mean`.
+
+Influencers:
+
+* `service.name` 
+* `trace.id`
+* `transaction.name` 
+--
+
+anomalous_error_rate_for_user_agents_jsbase::
+Detects user agents that are encountering errors at an above normal rate. This 
+can help detect browser compatibility issues.
++
+--
+Bucket span: 15m.
+
+Function: `high_non_zero_count`.
+
+Influencers:
+
+* `user_agent.name`
+* `error.exception.message.keyword`
+* `error.page.url`
+* `service.name`
+--
+
+decreased_throughput_jsbase::
+decreased_throughput_nodejs::
+Identifies periods during which the application is processing fewer requests 
+than normal.
++
+--
+Bucket span: 15m.
+
+Function: `low_count`.
+
+Influencers:
+
+* `service.name`
+* `transaction.name`
+--
+
+high_count_by_user_agent_jsbase::
+Detects user agents that are making requests at a suspiciously high rate. This 
+is useful in identifying bots.
++
+--
+Bucket span: 15m.
+
+Function: `high_non_zero_count`.
+
+Influencers:
+
+* `service.name`
+* `user_agent.name`
+--
+
+high_mean_response_time::
```

Comment:
While playing with an APM sample data set, this was the only job that was suggested for my data.  If possible, it would be nice to clarify which type of data / products / scenarios are required to have each of these wizards to appear.